packages/api/endpoints/granules.js

```diff
@@ -113,7 +115,15 @@ async function list(req, res) {
   } else {
     es = new Search({ queryStringParameters }, 'granule', process.env.ES_INDEX);
   }
-  const result = await es.query();
+  let result;
+  // TODO the condition should be removed after we support all the query parameters
```

Comment:
So is there one of these tickets that makes sense to add to the comment? If not, that's ok I guess, but I'm just thinking it might be helpful for whoever is working that last ticket to know this needs to be done with it.

Reply:
Added to [CUMULUS-3696](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3696)

Reply:
I think this will be my last comment. So I know this conditional will go away, but for now since we have it implemented I feel like there should be a test showing how this condition will work given different queries to the endpoint.

Reply:
The test I added tested the if statement.  If the query has only 'limit,page, sort_key`, it queries RDS, and returns 4 granules, otherwise, es has 3 records.

Reply:
> Added to [CUMULUS-3696](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3696)

Granule List endpoint has been updated here to query postgres.
https://github.com/nasa/cumulus/pull/3656/files#diff-624123a7dc5bb5513e3769b27fe85821577087609a9376829676b7edbeb3a5aeL119

I removed comment from CUMULUS-3696